Model/SnapshotPageProxyInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Sonata project.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\PageBundle\Model;
+
+/**
+ * SnapshotPageProxyInterface.
+ *
+ * @author Fabien D. <fabien@myprod.net>
+ */
+interface SnapshotPageProxyInterface extends PageInterface, \Serializable
+{
+}
```

Comment:
I've done that in order to test the type of the object in "Twig/Extension/PageExtension.php"